src/hotspot/share/gc/shenandoah/heuristics/shenandoahAdaptiveHeuristics.cpp

```diff
@@ -1,6 +1,7 @@
 /*
  * Copyright (c) 2018, 2019, Red Hat, Inc. All rights reserved.
  * Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Is it normal to update copyright header for changes such as this, i.e. for small cosmetic changes too? (In the past we got somewhat different feedback on this.)

Reply:
OK, thanks for checking, Coleen!